### PR TITLE
[AArch64][docs] Add release notes for FUJITSU-MONAKA support

### PR DESCRIPTION
**Pull request by kinoshita-fj**
Adds release notes for the FUJITSU-MONAKA support introduced in PR #118432. These notes were missing from the original PR.

### CODE REVIEW

File: clang/docs/ReleaseNotes.rst [hunk 1]
```diff
@@ -977,6 +977,10 @@ Arm and AArch64 Support
   in leaf functions after enabling ``-fno-omit-frame-pointer``, you can do so by adding
   the ``-momit-leaf-frame-pointer`` option.
 
+- Support has been added for the following processors (-mcpu identifiers in parenthesis):
```
Comment (ytmukai) [inline clang/docs/ReleaseNotes.rst:980]: It should be added that this is for AArch64 target.

Comment (kinoshita-fj) [reply clang/docs/ReleaseNotes.rst:980]: I've added in this commit 82164cd56dcd6a66a314124067c265d6293247ef.

**Review by davemgreen — APPROVED** [review]
LGTM, thanks

**Review by jthackray — APPROVED** [review]
LGTM

**Pull request merged**